src/signals/incident-management/routes.js

```diff
@@ -14,6 +14,7 @@ const routes = {
   defaultTexts: STANDARD_TEXT_URL_V1,
   standardTexts: STANDARD_TEXT_URL_V2,
   standardTextsDetail: `${STANDARD_TEXT_URL_V2}/:id`,
+  standardTextsDetailNew: `new`,
```

Comment:
What is this?

Reply:
That is the add-on to the url of standardTexts when you go to ad a new Standard Text.